conf.template

```diff
@@ -0,0 +1,6 @@
+[[annotation]]
+file="OUTPUT_DIR"
```

Comment:
shouldn't this be ANNOTATE_OUTPUT_DIR?

Reply:
this is the placholder that is replaced for the capice output with sed